packages/components/List/Item/styles.js

```diff
@@ -38,6 +38,14 @@ export const CheckboxStyled = styled(Checkbox)`
   }
 `;
 
+export const WebsiteFavIcon = styled.img`
+  width: 16px;
+  height 16px;
+  background-size: 16px;
+  background: url("https://www.google.com/s2/favicons?domain=${({ website }) =>
```

Comment:
This is dublicated string.
Let's create a const `https://www.google.com/s2/favicons?domain=`?

Reply:
The previous url only for the alt description. Also, it's good description for itself.